scripts/generate.sh

```diff
@@ -37,7 +37,7 @@ platform=$(uname)
 # clean models before generating
 rm -f ./src/Model/*
 
-java -jar ./bin/swagger-codegen-cli.jar generate -t ./template -l php -i ./res/fingerprint-server-api.yaml -o ./ -c config.json
+java -jar ./bin/swagger-codegen-cli.jar generate -t ./template -l php -i ./res/fingerprint-server-api.yaml -o ./ -c config.json --type-mapping RawDeviceAttributes=array,WebhookRawDeviceAttributes=array,Tag=array
```

Comment:
Can you explain this new flag and what we expect as an outcome?

Reply:
Sure. It replaces the provided types with the ones that we set up. So fir this change we will have `array` instead of `RawDeviceAttributes`